spec/transaction/transaction.yml

```diff
@@ -893,8 +945,66 @@ components:
             description: "Permission denied"
 # Note: this is defined as an overwrite in 'overwrites', since defining it here would violate
```

Comment:
This comment might be obsolete, right? Should I delete it?

Reply:
Jup